staging/src/k8s.io/kubectl/pkg/cmd/annotate/annotate_test.go

```diff
@@ -474,18 +474,57 @@ func TestAnnotateObject(t *testing.T) {
 		NegotiatedSerializer: resource.UnstructuredPlusDefaultContentConfig().NegotiatedSerializer,
 		Client: fake.CreateHTTPClient(func(req *http.Request) (*http.Response, error) {
 			switch req.Method {
-			case "GET":
+			case "GET", "PATCH":
 				switch req.URL.Path {
 				case "/namespaces/test/pods/foo":
 					return &http.Response{StatusCode: http.StatusOK, Header: cmdtesting.DefaultHeader(), Body: cmdtesting.ObjBody(codec, &pods.Items[0])}, nil
 				default:
 					t.Fatalf("unexpected request: %#v\n%#v", req.URL, req)
 					return nil, nil
 				}
-			case "PATCH":
+			default:
+				t.Fatalf("unexpected request: %s %#v\n%#v", req.Method, req.URL, req)
+				return nil, nil
+			}
+		}),
+	}
+	tf.ClientConfigVal = cmdtesting.DefaultClientConfig()
+
+	iostreams, _, bufOut, _ := genericclioptions.NewTestIOStreams()
+	cmd := NewCmdAnnotate("kubectl", tf, iostreams)
+	cmd.SetOut(bufOut)
+	cmd.SetErr(bufOut)
+	flags := NewAnnotateFlags(iostreams)
+	args := []string{"pods/foo", "a=b", "c-"}
+
+	options, err := flags.ToOptions(tf, cmd, args)
+	if err != nil {
+		t.Fatalf("unexpected error: %v", err)
+	}
+	if err := options.RunAnnotate(); err != nil {
+		t.Fatalf("unexpected error: %v", err)
+	}
+}
+
+func TestAnnotateObjectMultipleResources(t *testing.T) {
+	pods, _, _ := cmdtesting.TestData()
+
+	tf := cmdtesting.NewTestFactory().WithNamespace("test")
+	defer tf.Cleanup()
+
+	codec := scheme.Codecs.LegacyCodec(scheme.Scheme.PrioritizedVersionsAllGroups()...)
+
+	tf.UnstructuredClient = &fake.RESTClient{
+		GroupVersion:         schema.GroupVersion{Group: "testgroup", Version: "v1"},
```

Comment:
Do we need this line?  
```
GroupVersion: schema.GroupVersion{Group: "testgroup", Version: "v1"},
```

I don't think this unit test does anything with testgroup/v1 resources.

Same goes for line 473.

Reply:
Noted. Thank. I will update accordingly